templates/parts/credit.php

```diff
@@ -10,12 +10,14 @@
 <div class="site-info-text">
 <?php
 
-printf(
+$copyright_text = apply_filters( 'primer_copyright_text', get_theme_mod( 'copyright_text', sprintf(
```

Comment:
@EvanHerman Let's cast the value as a string and provide a docblock for the filter.